### PR TITLE
updating example app and augury to rc4

### PR DESCRIPTION
**Pull request by sumitarora**
Should fix: https://github.com/rangle/augury/issues/474

### CODE REVIEW

File: example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts [hunk 1]
```diff
@@ -31,10 +30,6 @@ import {UserInfoCheckOnce} from './user-info-checkonce';
     <user-info-push [user]="user">
     </user-info-push>
 
-    <user-info-checkonce [user]="user">
```
Comment (igor-ka) [inline example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts]: @sumitarora what is the reason behind removing this component

Comment (sumitarora) [inline example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts]: CheckOnce strategy has been removed with latest update https://angular.io/docs/ts/latest/api/core/index/ChangeDetectionStrategy-enum.html

Comment (igor-ka) [inline example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts]: 👍 thanks

**Comment by igor-ka** [comment]
@sumitarora LGTM :shipit:

File: example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts (hunk 1 above)
Comment (ericjim) [inline example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts]: Sadly, they haven't updated the stability status of the API to deprecated yet. 😭

**Pull request merged**